database/entity/0003-login_server_tables/LoginUser.ts

```diff
@@ -53,4 +54,7 @@ export class LoginUser extends BaseEntity {
 
   @Column({ name: 'publisher_id', default: 0 })
   publisherId: number
+
+  @OneToOne(() => LoginUserBackup, (loginUserBackup) => loginUserBackup.loginUser)
+  loginUserBackup: LoginUserBackup
```

Comment:
Good Point - I did not thought sbout this, since a One2One is an antipattern for me.
We could use this to refsctor slot of our code eritten for login/createUser etc. <3